<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>structure_customizer</class>
 <widget class="QDialog" name="structure_customizer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>71</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>71</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>71</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Structure Customizer</string>
  </property>
  <widget class="QPushButton" name="viewModeButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Stick Mode</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ballSizeSpinBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="stickSizeSpinBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply Changes</string>
   </property>
  </widget>
  <widget class="QLabel" name="ballSizeLabel">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>10</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ball Size:</string>
   </property>
  </widget>
  <widget class="QLabel" name="stickSizeLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Stick Size:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="toggleBondsButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Bonds Off</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
